mmv1/third_party/terraform/resources/resource_container_node_pool.go.erb

```diff
@@ -104,6 +104,25 @@ var schemaNodePool = map[string]*schema.Schema{
 		},
 	},
 
+<% unless version == 'ga' -%>
+	"placement_policy": {
+		Type:        schema.TypeList,
+		Optional:    true,
+		MaxItems:    1,
+		Description: `Specifies the node placement policy`,
+		Elem: &schema.Resource{
+			Schema: map[string]*schema.Schema{
+				"type": {
+					Type:         schema.TypeString,
+					Required:     true,
+					ValidateFunc: validation.StringInSlice([]string{"COMPACT", "TYPE_UNSPECIFIED"}, false),
```

Comment:
Would it ever be valid for a user to specify the type "TYPE_UNSPECIFIED"? If not, we may want to remove that type from this validate func.

If we expect there to be new values for placement policy types in the future we may want to not enforce this validation on the string itself and document the COMPACT type instead

Reply:
COMPACT and TYPE_UNSPECIFIED are both valid. Admittedly, TYPE_UNSPECIFIED is equivalent to not setting placement_policy, so from this point of view it's unlikely to  be used.

Unfortunately I can't say definitively for other values. Currently GCE supports also "spread" placement policies, but I don't know if GKE is going to support that.

Reply:
I would go with allowing only a COMPACT and leave it to GKE to silently set TYPE_UNSPECIFIED if no value provided.

Reply:
done